runtime-modules/content/src/lib.rs

```diff
@@ -2333,6 +2423,101 @@ impl<T: Trait> Module<T> {
         Ok(())
     }
 
+    fn execute_delete_video_mutation(
+        sender: &T::AccountId,
+        channel_id: T::ChannelId,
+        video_id: T::VideoId,
+        video: &Video<T>,
+        assets_to_remove: &BTreeSet<DataObjectId<T>>,
+    ) -> DispatchResult {
+        // bloat bond logic: channel owner is refunded
+        video
+            .video_post_id
+            .as_ref()
+            .map(|video_post_id| {
+                Self::video_deletion_refund_logic(&sender, &video_id, &video_post_id)
+            })
+            .transpose()?;
+
+        if !assets_to_remove.is_empty() {
+            Storage::<T>::delete_data_objects(
+                sender.clone(),
+                Self::bag_id_for_channel(&channel_id),
+                assets_to_remove.clone(),
+            )?;
+        }
+
+        // Remove video
+        VideoById::<T>::remove(video_id);
+
+        // Remove all comments related
+        <VideoPostById<T>>::remove_prefix(video_id);
+
+        // Update corresponding channel
+        // Remove recently deleted video from the channel
+        ChannelById::<T>::mutate(channel_id, |channel| {
+            channel.num_videos = channel.num_videos.saturating_sub(1)
+        });
+
+        Ok(())
+    }
+
+    fn ensure_channel_bag_can_be_dropped(
```

Comment:
Changed those functions to take `channel_id` instead of `dynamic_bag_id` in https://github.com/Joystream/joystream/pull/3366/commits/4aa4dac265b5e8a12d6e3428a2f16f30977175e3